Octokit.Tests.Integration/Helper.cs

```diff
@@ -50,9 +50,9 @@ public static class Helper
             return new Credentials(githubUsername, githubPassword);
         });
 
-        static readonly Lazy<Uri> _gitHubEnterpriseUrl = new Lazy<Uri>(() =>
+        static readonly Lazy<Uri> _customUrl = new Lazy<Uri>(() =>
         {
-            string uri = Environment.GetEnvironmentVariable("OCTOKIT_GITHUBENTERPRISEURL");
+            string uri = Environment.GetEnvironmentVariable("OCTOKIT_CUSTOMURL");
```

Comment:
:grey_question: I kinda liked having `ENTERPRISE` in this environment variable - could you tell me about what motivated this change?